javascript/core/array-methods/how-to-merge-two-arrays.md

```diff
@@ -63,41 +68,41 @@ console.log([1, 2] + [3, 4])
 Modify `array1` such that it contains `array2`'s contents in order.
 
 ```javascript
-var array1 = ['a', 'b', 'c'];
-var array2 = ['d', 'e', 'f'];
+let array1 = ['a', 'b', 'c'];
+let array2 = ['d', 'e', 'f'];
 
-Array.prototype.???
-.???(array1, array2);
+array1.???(???array2)
 ```
 
-* push
-* apply
-* merge
-* arrayList
-* on
-* after
-* concat
+* `push`
+* `...`
+* `merge`
+* `+`
+* `concat`
```

Comment:
backticks don't make any difference when displayed in the app. see photo

![image](https://user-images.githubusercontent.com/19802557/77750232-57577a00-701b-11ea-945b-60af6dbb947f.png)


```suggestion
* push
* ...
* merge
* +
* concat
```